tests/integration/snapshots/logs/async-metrics_python310.log

```diff
@@ -605,6 +606,7 @@ HTTP GET https://www.datadoghq.com/ Headers: ["Accept-Encoding:gzip, deflate","A
           "http.url": "XXXX.execute-api.eu-west-1.amazonaws.com",
           "http.url_details.path": "/httpapi/get",
           "http.method": "GET",
+          "http.route": "GET /httpapi/get",
```

Comment:
Should this http.route span tag include `GET`? Looking at the Datadog Agent implementation, it leaves out the method: https://github.com/DataDog/datadog-agent/commit/315c38f20f75a4a5ce90fcf691d10d3ed286966d .

Reply:
Good catch!